src/main/java/racingcar/InputHandler.java

```diff
@@ -0,0 +1,58 @@
+package racingcar;
+
+import camp.nextstep.edu.missionutils.Console;
+import java.util.Arrays;
+import java.util.List;
+
+public class InputHandler {
+    public List<String> readAndSplitCarNames() {
+        System.out.println("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)");
+        String input = Console.readLine();
+
+        validateCarNames(input);
+
+        return Arrays.asList(input.split(","));
+    }
+
+    public int readRoundNumber() {
+        System.out.println("시도할 회수는 몇 회인가요?");
+        String input = Console.readLine();
+
+        return validateAndParseRoundNumber(input);
+    }
+
+    public void validateCarNames(String input) {
+        if (input == null || input.isBlank()) {
+            throw new IllegalArgumentException("입력값이 비어있거나 공백 문자만 포함되어 있습니다.");
+        }
+
+        final String carNameRegex = "^[a-zA-Z0-9]{1,5}$";
+
+        for (String carName : input.split(",")) {
+            if (!carName.matches(carNameRegex)) {
+                throw new IllegalArgumentException("자동차 이름은 영어와 숫자만을 사용해서 만들 수 있고, 1자 이상 5자 이하만 가능합니다.");
+            }
+        }
+    }
+
+    public int validateAndParseRoundNumber(String input) {
```

Comment:
```suggestion
    public int validateAndParseRoundNumber(String input) throws IllegalArgumentException {
```

throws Exception을 통해서 명시적으로 Exception이 발생할 수 있는 지점이다라는 것을 보여주면서 작업할 수도 있을 것 같습니다!